src/integrationTest/java/org/openstreetmap/atlas/geography/atlas/raw/OsmPbfToSlicedRawAtlasTest.java

```diff
@@ -99,4 +104,31 @@ public void testPbfToSlicedRawAtlas()
                         + oldSlicedAtlas.numberOfAreas()
                         + oldSlicedAtlas.numberOfLines() == slicedRawAtlas.numberOfLines() - 1);
     }
+
+    @Test
+    public void testWaysSpanningOutsideOfCountry()
+    {
+        final Resource pbf = new InputStreamResource(
+                () -> OsmPbfLoaderIntegrationTest.class.getResourceAsStream("CUB_72-111.pbf"));
+        final CountryBoundaryMap map = new CountryBoundaryMap(
+                new InputStreamResource(() -> OsmPbfLoaderIntegrationTest.class
+                        .getResourceAsStream("CUB_osm_boundaries.txt.gz"))
+                                .withDecompressor(Decompressor.GZIP));
+        final SlippyTile tile = SlippyTile.forName("8-72-111");
+        final MultiPolygon boundary = map.countryBoundary("CUB").get(0).getBoundary();
+        final MultiPolygon loadingArea = tile.bounds().clip(boundary, ClipType.AND)
+                .getClipMultiPolygon();
+
+        final RawAtlasGenerator rawAtlasGenerator = new RawAtlasGenerator(pbf);
+        final Atlas rawAtlas = rawAtlasGenerator.build();
+
+        final Set<IsoCountry> countries = new HashSet<>();
+        countries.add(IsoCountry.forCountryCode("CUB").get());
+        final RawAtlasCountrySlicer slicer = new RawAtlasCountrySlicer(countries, map, loadingArea);
+
+        final Atlas slicedAtlas = slicer.slice(rawAtlas);
+
+        // Make sure that the big bridge over water made it to the Atlas
+        Assert.assertNotNull(slicedAtlas.line(308541861000000L));
```

Comment:
I believe that way is already inside the CUB boundary supplied for the test case, which will succeed, no matter if the code handles outside ways or not.